dobble/card.py

```diff
@@ -339,8 +339,9 @@ def generate_card(out_card_path: str,
 
         cropped_card_img[cropped_resized_mask] = cropped_resized_symbol[cropped_resized_mask]
 
-    cv2.circle(card_img, (card_size_pix // 2, card_size_pix // 2),
-               card_size_pix // 2, (0, 0, 0), circle_width_pix)
+    if circle_width_pix != 0:
```

Comment:
can you make it optional to explicit the fact that you want yo remove it?
and explain your "circle punch" trick in the doc ;)

Reply:
Sorry I don't get what your mean by "make it optional".
Do you want a new parameter rather than using card_size_pix?

Reply:
I was referring to the "Optional" type hint
I've added it in 6c1b014e40d66e4659cd785f779439f810773ff4

Reply:
Ok thanks 🙏